dist/spec.json

```diff
@@ -20272,12 +20272,12 @@
         "mapping": {
           "call": "#/definitions/CallUserFactor",
           "email": "#/definitions/EmailUserFactor",
-          "hotp": "#/definitions/CustomHotpUserFactor",
```

Comment:
Let's keep both to avoid breaking changes.

Reply:
In this case I'll need to add hotp to FactorType too, which is not correct, I think. Otherwise, okta-sdk-python won't work.
What do you think?

Reply:
It won't work if I don't add hotp to FactorType enum

Reply:
Oh yeah, you'll need to keep that as well. So, to summarize the changes, you shouldn't remove anything related to the existing `hotp` spec. You'll have to add the required changes to support `token:hotp` as well.